server/Dockerfile

```diff
@@ -33,6 +33,10 @@ RUN ./build-libraw.sh
 RUN ./build-imagemagick.sh
 RUN ./build-libvips.sh
 
+RUN sed -i -e's/ main/ main contrib non-free non-free-firmware/g' /etc/apt/sources.list.d/debian.sources
+COPY bin/install-ffmpeg.sh bin/build-lock.json ./
+RUN ./install-ffmpeg.sh
```

Comment:
These are in both dev and prod. Can you move this to another stage "builder" at the top and then have dev and prod both extend it?